simple-counter/src/lib.rs

```diff
@@ -84,7 +92,7 @@ mod tests {
     #[test]
     fn increment() {
         let auths = vec![accounts(0)];
-        let mut contract = State::new(10, auths);
+        let mut contract = State::default();
```

Comment:
`default()` 는 `new` 가 불리기 전에 컨트랙트가 가지고 있는 기본 상태입니다. 명시적으로 저희가 불러줄 필요도 없습니다. 

`new` 가 불려야 컨트랙트가 initilized 되며 시작되는 거기 때문에 원래대로 되돌려야합니다. (실제로 테스트가 전부 fail 나고요)

변경된 모든 라인 수정 부탁드립니다. 

```suggestion
        let mut contract = State::new(10, auths);
```

Reply:
수정하였습니다!